v2/internal/reconcilers/generic/register.go

```diff
@@ -133,7 +133,8 @@ func register(
 		WithOptions(options.Options)
 
 	for _, watch := range info.Watches {
-		builder = builder.Watches(watch.Src, watch.MakeEventHandler(kubeClient, options.LogConstructor(nil).WithName(info.Name)))
+		mgr.GetCache()
```

Comment:
What does `mgr.GetCache()` do and does it need to be called inside the loop repeatedly?

(`Get` reads as though it's an accessor for something, but you're not using any return value here.)

Reply:
That should've been removed. Thanks for pointing that out